phpcs.xml.dist

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0"?>
+<ruleset name="Core Sitemaps">
+
+	<config name="installed_paths" value="../../humanmade/coding-standards" />
+
+	<!-- Ignore directories we're not interested in -->
+	<exclude-pattern>/vendor/*</exclude-pattern>
+
+	<exclude-pattern>/tests/artifacts/*</exclude-pattern>
+	<exclude-pattern>/tests/app/www/*</exclude-pattern>
+	<exclude-pattern>/tests/bootstrap.php</exclude-pattern>
+
+	<!-- Ignore minified files -->
+	<exclude-pattern>*.js</exclude-pattern>
+	<exclude-pattern>*.min.js</exclude-pattern>
+	<exclude-pattern>*.min.css</exclude-pattern>
+
+	<rule ref="HM">
```

Comment:
As we're not using HM Coding Standards this rule shouldn't be needed

Reply:
I'll remove this block form the PR as there might not be a need for a WP equivalent.